gdpr/aggregated_config.go

```diff
@@ -10,13 +10,16 @@ import (
 // TCF2ConfigReader is an interface to access TCF2 configurations
 type TCF2ConfigReader interface {
 	BasicEnforcementVendor(openrtb_ext.BidderName) bool
+	BasicEnforcementVendors() map[string]struct{}
 	FeatureOneEnforced() bool
 	FeatureOneVendorException(openrtb_ext.BidderName) bool
 	IntegrationEnabled(config.IntegrationType) bool
 	IsEnabled() bool
 	PurposeEnforced(consentconstants.Purpose) bool
+	PurposeEnforcementAlgo(consentconstants.Purpose) string
 	PurposeEnforcingVendors(consentconstants.Purpose) bool
 	PurposeVendorException(consentconstants.Purpose, openrtb_ext.BidderName) bool
+	PurposeVendorExceptions(consentconstants.Purpose) map[openrtb_ext.BidderName]struct{}
```

Comment:
IMHO the difference of names between `BasicEnforcementVendor` and `BasicEnforcementVendors` is subtle. How are they related to each other?

Reply:
If one is returning a map and the other is returning one entry from that map, would it be better to just have the map returning function and just do the lookup locally to keep the interface cleaner?

Reply:
👍  I removed the `BasicEnforcementVendor` and `PurposeVendorException` functions from the interface and the various config objects and now we just have the interface functions that return the maps with lookups performed locally.